src/bttrx_wifi.cpp

```diff
@@ -164,9 +166,21 @@ void BTTRX_WIFI::setup(BTTRX_CONTROL *control)
 	}
 	bttrx_control_ = control;
 
+	Serial.println("");
 	Serial.println("Configuring access point...");
 
-	WiFi.softAP(WIFI_SSID, WIFI_PASSWORD);
+	String callsign = preferences.getString("callsign", "");
+	if (callsign != "") {
```

Comment:
Please use .empty() for this

Reply:
```
src/bttrx_wifi.cpp: In member function 'void BTTRX_WIFI::setup(BTTRX_CONTROL*)':
src/bttrx_wifi.cpp:173:16: error: 'class String' has no member named 'empty'
  if (!callsign.empty()) {
                ^
```
Same error as above. I will need to analyze further.

Reply:
We try to avoid usage of `String` wherever possible and try to use `string` instead (String != string).
This is because String is not supported by the gunit, so it would be a big effort to mock this class.
Therefore we decided to use the cpp string class instead.
So you should convert your `String callsign = ` form line 172 to string. Respectively, please implement a getter Method like `string BTTTRX_CONTROL::getCallsign()` where you call preferences.getString etc.

Reply:
Ok, got that. With change to string instead of String also .empty() is available. So this is fixed with https://github.com/phl0/firmware/commit/888408e3974973b4779f6f34db16dabc96b020a2.